lib/plug/csrf_protection.ex

```diff
@@ -133,10 +139,15 @@ defmodule Plug.CSRFProtection do
     ensure_csrf_token(conn, csrf_token)
   end
 
-  defp cross_origin_js?(conn) do
-    conn.method == "GET" and not xhr?(conn) and not plug_skip_csrf_protection?(conn) and
-      Enum.any?(get_resp_header(conn, "content-type"),
-                &String.starts_with?(&1, ["text/javascript", "application/javascript"]))
+  defp cross_origin_js?(%Plug.Conn{method: "GET"} = conn),
```

Comment:
:+1: for those changes but there is no need to add the parens around not (when we did not have it before).